packages/pluggableWidgets/range-slider-web/src/RangeSlider.xml

```diff
@@ -0,0 +1,202 @@
+                    <description />
+                    <returnType type="Decimal" />
+                </property>
+                <!-- Maximum Value End -->
+            </propertyGroup>
+            <propertyGroup caption="Steps">
+                <!-- Steps Size -->
+                <property key="stepSizeType" type="enumeration" defaultValue="static">
+                    <caption>Step size type</caption>
+                    <description />
+                    <enumerationValues>
+                        <enumerationValue key="static">Static</enumerationValue>
+                        <enumerationValue key="dynamic">Dynamic</enumerationValue>
+                        <enumerationValue key="expression">Expression</enumerationValue>
+                    </enumerationValues>
+                </property>
+                <property key="stepValue" type="decimal" defaultValue="1">
+                    <caption>Step size</caption>
+                    <description />
+                </property>
+                <property key="stepAttribute" type="attribute" required="false">
+                    <caption>Step size</caption>
+                    <description/>
+                    <attributeTypes>
+                        <attributeType name="Integer"/>
+                        <attributeType name="Long"/>
+                        <attributeType name="Decimal"/>
+                    </attributeTypes>
+                </property>
+                <property key="expressionStepSize" type="expression" required="false">
+                    <caption>Step size</caption>
+                    <description />
+                    <returnType type="Decimal" />
+                </property>
+                <!-- Steps Size End -->
+            </propertyGroup>
+            <propertyGroup caption="Tooltip">
+                <property key="showTooltip" type="boolean" defaultValue="true">
+                    <caption>Show tooltip</caption>
+                    <description />
+                </property>
+                <property key="tooltipTypeLower" type="enumeration" defaultValue="value">
+                    <caption>Lower bound tooltip type</caption>
+                    <description />
+                    <enumerationValues>
+                        <enumerationValue key="value">Value</enumerationValue>
+                        <enumerationValue key="customText">Custom</enumerationValue>
+                    </enumerationValues>
+                </property>
+                <property key="tooltipLower" type="textTemplate" required="false">
+                    <caption>Tooltip</caption>
+                    <description />
+                </property>
+                <property key="tooltipTypeUpper" type="enumeration" defaultValue="value">
+                    <caption>Upper bound tooltip type</caption>
+                    <description>By default tooltip shows current value. Choose 'Custom' to create your own template.</description>
```

Comment:
But both props in same place and In my view having two description is bit redundant.
If they will appear in two different places -- make sense, but in this case they one above other so...

Reply:
I think it's better to repeat. Now one has a description while the other ones doesn't which can be confusing. Especially since the comment is valid for both. We also don't know how it will look in Studio and whether they'll stay close to each forever.

Reply:
Yeah, good point about Studio